src/Files.App/Data/Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -718,6 +718,31 @@ public bool? IsHiddenEditedValue
 			set => SetProperty(ref isHiddenEditedValue, value);
 		}
 
+		private bool? isContentCompressed;
+		public bool? IsContentCompressed
```

Comment:
Could you add a comment like "This is only for NTFS" so others wouldn't be confused it with archive files.